src/main/resources/sql/bridge_participantoptions.sql

```diff
@@ -0,0 +1,5 @@
+CREATE TABLE IF NOT EXISTS bridge_participantoptions_<dateSuffix> (
+    healthDataCode      CHAR(36)        ENCODE RAW          NOT NULL,
+    studyKey            VARCHAR(50)     ENCODE BYTEDICT     NOT NULL,
+    data                VARCHAR(500)    ENCODE BYTEDICT     NOT NULL
```

Comment:
We will import the data as-is as the first step.  We can add any stuff in the data blob.  A query can still be executed against part of the data using `... WHERE data LIKE %ALL_QUALIFIED_RESEARCHERS%`.  But that is not efficient.  So as the second step of import, we want to derive a secondary table by parsing the JSON blob into additional columns.  A few feature for future work.